file_paths.md

```diff
@@ -208,111 +276,86 @@ create a `puppet` user or group.
-        puppetlabs.ico *                      # icon for start menu shortcut
-        puppetres.dll *                       # event log message resource dll
-        versions.txt *                        # versions of components
-
-    C:\Program Files\Puppet Labs\Puppet\puppet *
-        bin *
-            puppet *                          # ruby bin wrapper
-        lib *
-            puppet.rb *
 
-    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
-        bin *
-            pxp-agent.exe *
-            lib*.dll *                        # gcc, mingw, boost libraries
-                                              # pxp-agent also loads openssl libraries (below)
+        include
+            boost
+            cpp-pcp-client
+            curl
+            facter *                          # facter headers
+            leatherman
+            openssl
+            ruby-2.1.0
+
+        lib
+            cmake
+            engines
+            leatherman*.a
+            libcpp-pcp-client.so.a
+            libcrypto.a
+            libcrypto.dll.a
+            libcurl.dll.a
+            libcurl.la
+            libfacter.so.a
+            libpxp-agent.a
+            libssl.a
+            libssl.dll.a
+            libx64-msvcrt-ruby210.dll.a
+            libx64-msvcrt-ruby210-static.a
+            pkgconfig
+                libcrypto.pc
+                libcurl.pc
+                libssl.pc
+                openssl.pc
+                ruby-2.1.pc
+            ruby
+                2.1.0
+                gems
+                    2.1.0
+                        specifications
+                            facter.gemspec
+                            hiera.gemspec
+                            puppet.gemspec
+                vendor_ruby
+                    facter.rb *               # facter ruby bindings
+                    hiera.rb
+                    hiera/*
+                    hiera_puppet.rb
+                    puppet.rb
+                    puppet/*
+                    mcollective.rb
+                    mcollective/*
+                    puppet.rb
+                    puppet/*
+                    puppet_x.rb
+                    semver.rb
         modules *
-            pxp-module-puppet *
-            pxp-module-puppet.bat *
-
-    C:\Program Files\Puppet Labs\Puppet\service *
-        daemon.rb *                           # windows service daemon
-        nssm.exe *                            # NSSM used to run pxp-agent
-
-    C:\Program Files\Puppet Labs\Puppet\sys *
-        ruby *
-            bin *
-                ruby.exe *
-                ssleay32.dll *                # openssl dll
-                libeay32.dll *                # openssl dll
-                lib*.dll *                    # yaml, iconv, gdbi & ffi libraries
-            include *
-            lib *
-            share *
-
-        tools *
-            bin *
-              elevate.exe *                   # Used to elevate interactive commands
```

Comment:
We should probably add a new component file for this, unless it can be categorized under ruby or something else. But yeah, we can totes drop it into buildsources and make sure `elevate.exe.config` is in the same directory